tests/test-template-tags.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+class Test_Template_Tags_CoAuthors extends CoAuthorsPlus_TestCase {
+
+	/**
+	 * Test get Co-Authors function
+	 */
+	public function test_is_coauthor_for_post() {
+
+		// Test only sending user
+		$this->assertEquals( false, is_coauthor_for_post( '' ) );
```

Comment:
Why did you decide to have many asserts in the same method, instead of separate methods for each check?